src/components/UserList/index.tsx

```diff
@@ -149,16 +183,178 @@ const UserList: React.FC = () => {
           {intl.formatMessage(messages.deleteconfirm)}
         </Modal>
       </Transition>
-      <div className="flex items-center justify-between">
-        <Header>{intl.formatMessage(messages.userlist)}</Header>
-        <Button
-          className="mx-4 my-8"
-          buttonType="primary"
-          disabled={isImporting}
-          onClick={() => importFromPlex()}
+      <Transition
+        enter="opacity-0 transition duration-300"
+        enterFrom="opacity-0"
+        enterTo="opacity-100"
+        leave="opacity-100 transition duration-300"
+        leaveFrom="opacity-100"
+        leaveTo="opacity-0"
+        show={createModal.isOpen}
+      >
+        <Modal
+          title={intl.formatMessage(messages.createuser)}
+          iconSvg={<AddUserIcon className="h-6" />}
```

Comment:
This modal needs `onCancel` and `onPrimary` instead of rendering the buttons yourself. Look at other uses of the modal component. (Also as a side note, the create button should be button style of primary, as well)